src/citrine/informatics/design_candidate.py

```diff
@@ -159,9 +159,6 @@ class HierarchicalDesignMaterial(Serializable["HierarchicalDesignMaterial"]):
     mixtures = properties.Mapping(properties.UUID, properties.Object(Mixture), 'mixtures')
     """:Dict[UUID, Mixture]: mapping from Citrine ID to components the material is composed of"""
 
-    def __init__(self):
```

Comment:
I removed all the __init__ in this file that didn't do anything. Not sure if you wanted all of em removed. 🤷  But tests still pass. 😆